### ISSUE TITLE
Fails to parse FROM statement with database, but no schema specified

### ISSUE THREAD

**Issue by pwildenhain** [alanmcruickshank/sqlfluff#206]
We write a lot of SQL that assumes a default schema set for the user. For example:
```sql
select
    table_1.field
from my_db..table_1
```
However, this fails to parse:
```
$ sqlfluff lint test.sql
== [test.sql] FAIL
L:   3 | P:   1 |  PRS | Found unparsable segment @L003P001: 'from my_db..table_1\n...'
```
If I add the schema, it works as expected, the following produces no lint
```sql
select
    table_1.field
from my_db.my_schema.table_1
```

Here's a little more info about this type of notation: https://stackoverflow.com/questions/34786049/double-dot-table-qualifier

**Comment by alanmcruickshank** [comment]
Thanks for the post! I wasn't aware of this syntax, I'm happy to support it though :)
